src/arch/src/aarch64/regs.rs

```diff
@@ -151,12 +172,11 @@ arm64_sys_reg!(MIDR_EL1, 3, 0, 0, 0, 0);
 ///
 /// # Arguments
 ///
-/// * `state` - Array slice of kvm_one_reg structures, representing
-///             the registers of a VCPU state.
-pub fn get_manufacturer_id_from_state(state: &[kvm_one_reg]) -> Result<u32> {
+/// * `state` - Array slice of kvm_one_reg structures, representing the registers of a VCPU state.
```

Comment:
Nit: comment is out of date

Reply:
this should probably also get fixed on main - I'll make a note to do a follow up PR with some cleanup once I have the time